src/org/intellij/erlang/debugger/xdebug/ErlangXDebugProcess.java

```diff
@@ -358,6 +361,25 @@ private OSProcessHandler runDebugTarget() throws ExecutionException {
     return erlangProcessHandler;
   }
 
+  private void setupErlangAppConfig(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isLoadingConfig()) {
+      commandLine.addParameters("-config", runConfig.getDebugOptions().getAppConfig());
+    }
+  }
+
+  private void setupErlangRebarDependencies(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isIncludingRebarDependencies()) {
+      Set<String> dependencies = RebarConfigUtil.getRebarDependencies(
+              runConfig.getConfigurationModule().getModule(),
+              runConfig.getDebugOptions().isFetchingDependencies());
+
+      for (String dep : dependencies)
+        commandLine.addParameters("-pa", dep);
+    }
```

Comment:
So if the problem is in project importer, we should fix the importer and not put hacks to run configuration logic.

Could you share an example of a project where import fails and code path is not set up correctly?